Sources/Workspace/InitPackage.swift

```diff
@@ -269,6 +281,25 @@ public final class InitPackage {
                                 ]),
                         ]
                     """
+                } else if packageType == .buildToolPlugin {
+                    param += """
+                            .plugin(
+                                name: "\(typeName)",
+                                capability: .buildTool()
+                            ),
+                        ]
+                    """
+                } else if packageType == .commandPlugin {
+                    param += """
+                            .plugin(
+                                name: "\(typeName)",
+                                capability: .command(intent: .custom(
+                                    verb: "\(typeName)",
```

Comment:
does this work with spaces, etc?

Reply:
Yes.  This is the module name, which has already been mangled.